### PR TITLE
g*.py: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/gandi_livedns.py
plugins/modules/gconftool2.py
plugins/modules/gconftool2_info.py
plugins/modules/gem.py
plugins/modules/gio_mime.py
plugins/modules/git_config.py
plugins/modules/git_config_info.py
plugins/modules/github_deploy_key.py
plugins/modules/github_issue.py
plugins/modules/github_key.py
plugins/modules/github_release.py
plugins/modules/github_repo.py
plugins/modules/github_webhook.py
plugins/modules/github_webhook_info.py
plugins/modules/gitlab_branch.py
plugins/modules/gitlab_deploy_key.py
plugins/modules/gitlab_group.py
plugins/modules/gitlab_group_access_token.py
plugins/modules/gitlab_group_members.py
plugins/modules/gitlab_group_variable.py
plugins/modules/gitlab_hook.py
plugins/modules/gitlab_instance_variable.py
plugins/modules/gitlab_issue.py
plugins/modules/gitlab_label.py
plugins/modules/gitlab_merge_request.py
plugins/modules/gitlab_milestone.py
plugins/modules/gitlab_project.py
plugins/modules/gitlab_project_access_token.py
plugins/modules/gitlab_project_badge.py
plugins/modules/gitlab_project_members.py
plugins/modules/gitlab_project_variable.py
plugins/modules/gitlab_protected_branch.py
plugins/modules/gitlab_runner.py
plugins/modules/gitlab_user.py
plugins/modules/grove.py
plugins/modules/gunicorn.py

### CODE REVIEW

**Comment by guenhter** [comment]
LGTM

**Comment by gpongelli** [comment]
LGTM

File: plugins/modules/gandi_livedns.py [hunk 1]
```diff
@@ -61,25 +59,25 @@
...
-    - The record values.
-    - Required when O(state=present).
+      - The record values.
+      - Required when O(state=present).
     type: list
     elements: str
   domain:
     description:
-    - The name of the Domain to work with (for example, "example.com").
+      - The name of the Domain to work with (for example, "example.com").
```
Comment (felixfontein) [inline plugins/modules/gandi_livedns.py:68]:
```suggestion
      - The name of the Domain to work with (for example, V(example.com)).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for your contribution!
@guenhter @gpongelli thanks for reviewing!

**Pull request merged**